File: python/1038.py
item, quantity = input().split()
itemCod = int(item)
quantity = int(quantity)

itens = [{"item": 'Cachorro Quente', "price": 4.00},
         {"item": 'X-Salada', "price": 4.50},
         {"item": 'X-Bacon', "price": 5.00},
         {"item": 'Torrada simples', "price": 2},
         {"item": 'Refrigerante', "price":1.50}]

print(f'Total: R$ {(itens[itemCod - 1]["price"] * quantity):.2f}')

File: lua/1038.lua
local products = {
    [1] = {"Cachorro Quente", 4.00},
    [2] = {"X-Salada", 4.50},
    [3] = {"X-Bacon", 5.00},
    [4] = {"Torrada simples", 2.00},
    [5] = {"Refrigerante", 1.50},
}

local entry = io.read()
local t = {}

for v in string.gmatch(entry, "[^%s]+") do
    table.insert(t, v)
end

local id = tonumber(t[1])
local qty = tonumber(t[2])

local total = products[id][2] * qty

print("Total: R$ "..string.format("%.2f", total))